README.md

```diff
@@ -152,6 +152,26 @@ colorscheme onedark
 
 Example custom colors and Highlights config
 
+```lua
+require('onedark').setup {
+  colors = {
+    bright_orange = "#ff8800",    -- define a new color
+    green = '#00ffaa',            -- redefine an existing color
+  },
+  highlights = {
+    ["@keyword"] = {fg = '$green'},
+    ["@string"] = {fg = '$bright_orange', bg = '#00ff00', fmt = 'bold'},
+    ["@function"] = {fg = '#0000ff', sp = '$cyan', fmt = 'underline,italic'},
+    ["@function.builtin"] = {fg = '#0059ff'}
+  }
+}
+```
+Note that TreeSitter keywords have been changed after neovim version 0.8 and onwards.
+TS prefix is trimmed and lowercase words should be used separated with '.'
+
+The old way before neovim 0.8 looks like this.
+For all keywords see [this](https://github.com/navarasu/onedark.nvim/blob/master/lua/onedark/highlights.lua) file from line 133 to 257
```

Comment:
I would suggest changing that link to either https://github.com/navarasu/onedark.nvim/blob/master/lua/onedark/highlights.lua#L133-L257 to always point to the current file or https://github.com/navarasu/onedark.nvim/blob/fdfe7bfff486acd102aae7fb2ff52e7e5f6c2bad/lua/onedark/highlights.lua#L133-L257 to always point to this specific commit.